Assignment6.Rmd

```diff
@@ -66,4 +68,13 @@ table(M2$certified, M2$predict1)
 
 table(M2$certified, M2$predict2)
 
```

Comment:
Do you have an explanation for your analysis? Do you think one of your models was overfitting?